app/screens/welcome-screens/welcome-screens.tsx

```diff
@@ -1,100 +0,0 @@
-/* eslint-disable react-native/no-color-literals */
-/* eslint-disable react-native/no-unused-styles */
-import { StackNavigationProp } from "@react-navigation/stack"
-import * as React from "react"
-import { Text, View } from "react-native"
-import EStyleSheet from "react-native-extended-stylesheet"
-import Swiper from "react-native-swiper"
-import { OnboardingScreen } from "../../components/onboarding"
-import { Screen } from "../../components/screen"
```

Comment:
The `welcomeFirst` field should be removed from `RootStackParamList` on line 20 of app/navigation/stack-param-lists.ts